### ISSUE TITLE
Missing hash-key: "give-url" (KeyError) when trying to run with only album & artist

### ISSUE THREAD

**Issue by bogga** [cooperhammond/irs#70]
```
Unhandled exception: Missing hash key: "give-url" (KeyError)
  from ../../usr/share/crystal/src/hash.cr:1027:11 in '[]'
  from docs/irs/src/bottle/cli.cr:95:19 in 'act_on_args'
  from docs/irs/src/irs.cr:5:3 in 'main'
  from docs/irs/src/irs.cr:8:1 in '__crystal_main'
  from ../../usr/share/crystal/src/crystal/main.cr:105:5 in 'main_user_code'
  from ../../usr/share/crystal/src/crystal/main.cr:91:7 in 'main'
  from ../../usr/share/crystal/src/crystal/main.cr:114:3 in 'main'
  from __libc_start_main
  from _start
  from ???
```
Ran this `irs --album "Demon Days" --artist "Gorillaz"`

No error thrown when running `irs --song "Bohemian Rhapsody" --artist "Queen"`

**Comment by cooperhammond** [comment]
Should be fixed with #70! Sorry for the slow response to such a simple bug, as usual, school continues to kick my ass.

**Comment by bogga** [comment]
No problem, I hope school is going well!
